### PR TITLE
Switch parameters to YAML

### PR DESCRIPTION
**Pull request by michelbl**
Connected to https://github.com/openfisca/openfisca-core/issues/523

## 1.2.4 - [#17](https://github.com/openfisca/country-template/pull/17)

* Technical improvement: adapt to version `17.0.0` of Openfisca-Core
* Details:
  - Transform XML parameter files to YAML parameter files.

### CODE REVIEW

File: openfisca_country_template/parameters/_.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# see https://doc.openfisca.fr/parameters.html
```
Comment (fpagnoux) [inline openfisca_country_template/parameters/_.yaml]: Do we really need this almost empty file ?

File: openfisca_country_template/parameters/benefits/_.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+type: node
```
Comment (fpagnoux) [inline openfisca_country_template/parameters/benefits/_.yaml]: Same question here

File: openfisca_country_template/parameters/taxes/income_tax_rate.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+description: Income tax rate
+type: parameter
+values:
+  '2012-01-01':
```
Comment (fpagnoux) [inline openfisca_country_template/parameters/taxes/income_tax_rate.yaml]: Are these quotes necessary ?

File: openfisca_country_template/reforms/modify_social_security_taxation.py [hunk 4]
```diff
@@ -14,30 +14,35 @@ class modify_social_security_taxation(Reform):
...
-        # This function takes an argument `reference_legislation_json_copy` which is a JSON-like representation
-        # of the XML element. It can be modified and must be returned.
+    def modify_brackets(self, legislation):
+        # This function takes an argument `legislation` which is a in-memory representation
+        # of the YAML parameters. It can be modified and must be returned.
 
         # Access the right legislation node:
-        scale = reference_legislation_json_copy['children']['taxes']['children']['social_security_contribution']
-        brackets = scale['brackets']
+        brackets = legislation.taxes.social_security_contribution.brackets
```
Comment (fpagnoux) [inline openfisca_country_template/reforms/modify_social_security_taxation.py]: > 🎉

File: openfisca_country_template/parameters/_.yaml (hunk 1 above)
Comment (michelbl) [inline openfisca_country_template/parameters/_.yaml]: discussed with Anna-Livia and Matti

File: openfisca_country_template/parameters/benefits/_.yaml (hunk 2 above)
Comment (michelbl) [inline openfisca_country_template/parameters/benefits/_.yaml]: removed

File: openfisca_country_template/parameters/taxes/income_tax_rate.yaml (hunk 3 above)
Comment (michelbl) [inline openfisca_country_template/parameters/taxes/income_tax_rate.yaml]: removed

File: openfisca_country_template/reforms/modify_social_security_taxation.py (hunk 4 above)
Comment (michelbl) [inline openfisca_country_template/reforms/modify_social_security_taxation.py]: :smile:

**Pull request merged**